vendor/README.md

```diff
@@ -572,6 +571,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **UnrealScript:** [textmate/java.tmbundle](https://github.com/textmate/java.tmbundle)
 - **UrWeb:** [gwalborn/UrWeb-Language-Definition](https://github.com/gwalborn/UrWeb-Language-Definition)
 - **V:** [0x9ef/vscode-vlang](https://github.com/0x9ef/vscode-vlang)
+- **VBA:** [serkonda7/vscode-vba](https://github.com/serkonda7/vscode-vba)
```

Comment:
You shouldn't need to make this change yourself... you're only changing the source of the SourcePawn grammar, not anything else.

Reply:
NM. I see I'm looking at old commits.